presto-main/src/test/java/io/prestosql/operator/aggregation/minmaxby/TestMinMaxByAggregation.java

```diff
@@ -86,7 +86,7 @@ public void testMinUnknown()
                 null,
                 createBooleansBlock(null, null),
                 createDoublesBlock(1.0, 2.0));
-        InternalAggregationFunction unknownValue = METADATA.getAggregateFunctionImplementation(METADATA.resolveFunction(QualifiedName.of("min_by"), fromTypes(DOUBLE, UNKNOWN)));
+        METADATA.getAggregateFunctionImplementation(METADATA.resolveFunction(QualifiedName.of("min_by"), fromTypes(DOUBLE, UNKNOWN)));
```

Comment:
remove call to `getAggregateFunctionImplementation`

Reply:
Thanks, remove call already.